code/controllers/subsystem/traumas.dm

```diff
@@ -69,7 +69,8 @@ SUBSYSTEM_DEF(traumas)
 												 /obj/item/trash/cheesie)),
 
 					  "greytide"  = typecacheof(list(/obj/item/clothing/under/color/grey, /obj/item/melee/baton/cattleprod,
-												 /obj/item/twohanded/spear, /obj/item/clothing/mask/gas)),
+												 /obj/item/twohanded/spear, /obj/item/clothing/mask/gas, /obj/item/clothing/mask/gas/alt, /obj/item/clothing/mask/gas/alt2,
```

Comment:
they dont strike the same fear as the old one

Reply:
ok good point